### PR TITLE
Remove thirdweb

### PR DESCRIPTION
**Pull request by sirgawain0x**

### CODE REVIEW

**Comment by sirgawain0x** [comment]
@greenealexander would you mind taking a look. We are removing thirdweb for soley wagmi and connectkit
Screenshot 2023-01-19 at 4 28 38 PM

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -9,25 +9,22 @@ import { Seo } from 'components/layout/Seo'
...
-    </ThirdwebProvider>
+    <ChakraProvider>
+      <Seo />
+      <Web3Provider>
+        {isMounted && (
+          <Layout>
+            <QueryClientProvider client={queryClient}>
+              <Component {...pageProps} />
+            </QueryClientProvider>
+          </Layout>
```
Comment (greenealexander) [inline src/pages/_app.tsx]: @gawainb `SignInButton` is in `Header` which is in `Layout`, but `Layout` is not a child of `QueryClientProvider`

```suggestion
          <QueryClientProvider client={queryClient}>
            <Layout>
              <Component {...pageProps} />
            </Layout>
          </QueryClientProvider>
```

**Comment by sirgawain0x** [comment]
@greenealexander and @starhelios Is it possible to use SIWE to sign in to Lens? 

The authflow  is:
1 connect wallet
2. SIWE by validating user has NFT
3. Sign in Lens

**Comment by greenealexander** [comment]
@gawainb PR #14 contains changes to automatically try to login to lens when their wallet is connected

**Review by starhelios — APPROVED** [review]

**Pull request merged**